api/src/wfl/tsv.clj

```diff
@@ -77,7 +77,7 @@
   [table]
   (let [writer (StringWriter.)]
     (write table writer)
-    (.toString writer)))
+    (str writer)))
```

Comment:
Kinda prefer `.toString` here as well, but OK if that works.
(I know some linter complains.)